python/change-notes/2021-03-12-small-api-enhancements.md

```diff
@@ -0,0 +1,5 @@
+lgtm,codescanning
+* Make `ParameterNode` extend `LocalSourceNode`, thus making members like `flowsTo` available.
+* Add member predicate `taintFlowsTo` to `LocalSourceNode` facilitating smoother syntax for local taint tracking.
+* Add member `getALocalTaintSource` to `DataFlow::Node` facilitating smoother syntax for local taint tracking.
+* Add predicate `parameterNode` to map from a `Parameter` to a data-flow node.
```

Comment:
I'm a bit unsure about the way in which these sentences are formulated. They read more like commit messages than change notes. Perhaps one could rewrite this as e.g.

- The class `ParameterNode` now extends `LocalSourceNode`, thus making methods like `flowsTo` available.
- Local taint tracking can now be performed using the `taintFlowsTo` method on the `LocalSourceNode` class.

and so on...

Reply:
I like that direction..